standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestHiveMetaStore.java

```diff
@@ -3717,4 +3683,42 @@ public void testDropDataConnectorIfNotExistsTrue() throws Exception {
     // No such data connector, ignore NoSuchObjectException
     client.dropDataConnector("no_such_data_connector", true, false);
   }
+
+  @Test
+
+  public void testIfFSWritesIsSkipped() throws Throwable {
+    skipFSWritesTester();
```

Comment:
seems we do not need introduce another method to perform tests

Reply:
Yes. The extra method is removed.